connector/spanmetricsconnector/connector.go

```diff
@@ -121,27 +99,48 @@ func newDimensions(cfgDims []Dimension) []dimension {
 }
 
 func newConnector(logger *zap.Logger, config component.Config, ticker *clock.Ticker) (*connectorImp, error) {
-	logger.Info("Building spanmetrics")
-	pConfig := config.(*Config)
-
-	bounds := defaultLatencyHistogramBucketsMs
-	if pConfig.LatencyHistogramBuckets != nil {
-		bounds = mapDurationsToMillis(pConfig.LatencyHistogramBuckets)
-	}
+	logger.Info("Building spanmetrics connector")
+	cfg := config.(*Config)
 
-	metricKeyToDimensionsCache, err := cache.NewCache[metricKey, pcommon.Map](pConfig.DimensionsCacheSize)
+	metricKeyToDimensionsCache, err := cache.NewCache[string, pcommon.Map](cfg.DimensionsCacheSize)
 	if err != nil {
 		return nil, err
 	}
 
+	var histograms metrics.HistogramMetrics
+	if cfg.Histogram.Exponential != nil {
```

Comment:
It looks like the intent is for Exponential xor Explicit to be configured (not both). What do you think about having an explicit check if both are `!= nil` and return an error to the user just to avoid confusion in the resulting behaviour?

Reply:
Sure, I'll do that in the configuration validation.